internal/cmd/info.go

```diff
@@ -36,6 +36,14 @@ func info(cli *CLI) error {
 		return err
 	}
 
+	user, err := client.GetUser(config.UserID)
+	if err != nil {
+		if api.ErrorStatusCode(err) == 401 {
+			return Error{Message: "Session is not valid for this server; run 'infra login' to start a new session"}
```

Comment:
It seems like this would happen for any API call on a new server (ex: If I installed a new server and ran `infra list`). Any way we can check for this scenario in the CLI error handling logic?

Reply:
I thought about this too, but I don't see an easy way. 401s means different things for each cmd - EG)
1. Info - unauthorized means they don't have access to the server, otherwise they would have been able to query OR fail as not logged in/expired. 
2. Destinations list - unauthorized means they don't have enough access to query the list of destinations
3. users edit - unauthorized means they are trying to edit someone else and they are not admin

All three 401s means different things, so I wasn't able to centralize this in a catch-all, like NotLoggedIn (CLI) or BadRequest (API).